uber/configspec.ini

```diff
@@ -49,6 +49,9 @@ at_the_con = boolean(default=False)
 # and behaviors such as automated emails which are ONLY turned on afterwards.
 post_con = boolean(default=False)
 
```

Comment:
I don't have any strong opinions on whether there should be a setting which turns off all internet-dependent stuff.  Like I think it's fine if this is a setting which only controls whether or not kiosks have the ability to make CC payments.  I was only bringing up the other stuff because the name of the setting (``offline_mode``) and the comment ("disables any internet-dependent features") led me to believe that your intention was to make it disable all that other stuff as well.

If we want to keep the implementation limited to just affecting at-the-door CC payments, I'd recommend renaming it to something like ``kiosk_cc_enabled`` or something along those lines.